src/reference/asciidoc/gateway.adoc

```diff
@@ -175,6 +175,33 @@ However, default headers do NOT override any `@Header` annotations in the servic
 
 The gateway now also supports a `default-payload-expression`, which is applied for all methods (unless overridden).
 
+Starting with version 5.2, the invoked gateway `Method` is always presented as a `gatewayMethod` message header, alongside with the `gatewayArgs` header for the `Object[]` with actually method invocation arguments.
```

Comment:
s/presented/added/
s/alongside.../along with the `gatewayArgs` header with the value being the actual method invocation arguments in an `Object[]`./